cmd/machine-healthcheck/main.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/golang/glog"
 	mapiv1 "github.com/openshift/cluster-api/pkg/apis/machine/v1beta1"
 	"github.com/openshift/machine-api-operator/pkg/apis/healthchecking/v1alpha1"
-	"github.com/openshift/machine-api-operator/pkg/controller"
+	"github.com/openshift/machine-api-operator/pkg/controllers"
```

Comment:
`github.com/openshift/machine-api-operator/pkg/controllers/<CONTROLLERNAME>` does not make much sense since you are importing a specific controller, not controllers. Although, it's always a matter of convention. And the convention has been to use singular form instead of the plural one. Please, drop this commit unless you have reasonable justification why is it better over the singular form.

Reply:
I just thought that `controllers` makes more sense because this package includes two controllers now, but I do not have strong justification except it, so will roll it back

Reply:
done